build.gradle

```diff
@@ -49,7 +49,31 @@ def includeDesktopSupport = false
 
 // Defining my dependencies. In this case, WPILib (+ friends), and vendor libraries.
 // Also defines JUnit 5.
+// repositories {
+//     maven {
+//         url = uri("https://maven.pkg.github.com/Mechanical-Advantage/AdvantageKit")
+//         credentials {
+//             username = "Mechanical-Advantage-Bot"
+//             password = "\u0067\u0068\u0070\u005f\u006e\u0056\u0051\u006a\u0055\u004f\u004c\u0061\u0079\u0066\u006e\u0078\u006e\u0037\u0051\u0049\u0054\u0042\u0032\u004c\u004a\u006d\u0055\u0070\u0073\u0031\u006d\u0037\u004c\u005a\u0030\u0076\u0062\u0070\u0063\u0051"
+//         }
+//     }
+//     mavenLocal()
+// }
+
+// configurations.all {
+//     exclude group: "edu.wpi.first.wpilibj"
+// }
+
+// task(checkAkitInstall, dependsOn: "classes", type: JavaExec) {
+//     mainClass = "org.littletonrobotics.junction.CheckInstall"
+//     classpath = sourceSets.main.runtimeClasspath
+// }
+// compileJava.finalizedBy checkAkitInstall
 dependencies {
+    // ...
+    // def akitJson = new groovy.json.JsonSlurper().parseText(new File(projectDir.getAbsolutePath() + "/vendordeps/AdvantageKit.json").text)
+    // annotationProcessor "org.littletonrobotics.akit.junction:junction-autolog:$akitJson.version"
```

Comment:
Let's go ahead and remove these commented out lines.

Reply:
done